.github/workflows/r-ci.yml

```diff
@@ -82,7 +99,7 @@ jobs:
       #  run: cd apis/r && Rscript -e "options(bspm.version.check=TRUE); install.packages('tiledb', repos = c('https://eddelbuettel.r-universe.dev/bin/linux/jammy/4.3/', 'https://cloud.r-project.org'))"
 
       - name: Dependencies
-        run: cd apis/r && tools/r-ci.sh install_all
+        run: cd apis/r && Rscript -e "remotes::install_deps(dependencies = TRUE, upgrade = FALSE)"
```

Comment:
@mojaveazure should this be
```
remotes::install_deps(".", dependencies = TRUE, upgrade = FALSE)"
```
?

Reply:
Not necessarily, the `"."` is the default https://remotes.r-lib.org/reference/install_deps.html